.github/workflows/reusable-docker.yml

```diff
@@ -35,6 +35,10 @@ on:
       env_tag:
         required: false
         type: string
+      community_build:
+        required: false
+        default: false
+        type: boolean
```

Comment:
this file should be adjusted based on this argument, the same thing, we don't need to build the enterprise packages